index.js

```diff
@@ -69,11 +69,6 @@ module.exports = function prettyFactory (options) {
       log = inputData
     }
 
-    // Short-circuit for spec allowed primitive values.
-    if ([null, true, false].includes(log) || Number.isFinite(log)) {
-      return `${log}\n`
-    }
-
```

Comment:
Why remove this? It's checking for the input being literally `null`, `true`, `false`, or a number and simply passing it along to the output. It's covered by these tests https://github.com/pinojs/pino-pretty/blob/ec3c5854a4b42f9fd9005bfa4a94756e484f8732/test/basic.test.js#L354-L386

Reply:
Because it's never triggered. It's currently handled in https://github.com/pinojs/pino-pretty/blob/ec3c5854a4b42f9fd9005bfa4a94756e484f8732/index.js#L65